source/_integrations/select.markdown

```diff
@@ -11,16 +11,25 @@ ha_codeowners:
 ha_integration_type: entity
 ---
 
-Keeps track on `select` entities in your environment, their state, and allows
+The **Select** {% term integration %} keeps track of the `select` {% term entities %} in your environment, their state, and allows
```

Comment:
Side note: I think we should rewrite all these paragraphs that say "keeps track of entities in your environment".

If environment is meant as a computer environment it's a too technical term used by developers and not for users. If it's meant as the home environment it's weird to talk about entities in that environment since they are not a real thing.

I'd just write:

> The select integration manages the state of the select entities and allows you to control them.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/34809